source/usage-examples/deleteMany.txt

```diff
@@ -33,7 +33,7 @@ collection in the ``sample_mflix`` database.
 
 The query filter passed to the ``deleteMany()`` method matches all
 movie documents that contain a ``rating`` of less than **1.9** in the ``imdb``
-sub-document.
+child document.
```

Comment:
Issue:

I don't think "child" is one of the designations we use for this object structure. The common term across our docs (and even outside our docs) is usually one of the following:
- subdocument
- embedded document
- nested document

While not necessarily "official", here's a [response](https://www.mongodb.com/community/forums/t/embedded-document-or-subdocument/159377/4) to a related question from Stennie (formerly DevRel) on our forums:

> `“Embedded document” and “subdocument” are used interchangeably, and you’ll also see references like “embedded subdocument” and “nested document”. Preferred naming conventions also depend on context (for example, language drivers or ODMs can be influenced by the author or phrases that are more idiomatic for a language community).


Suggestion:
Use one of the common terms, favoring consistency among the Java docs and then our other docs.

Applies to all instances of this.